server/storage/rethinkdb.go

```diff
@@ -87,35 +111,27 @@ func NewRethinkDBStorage(dbName, user, password string, sess *gorethink.Session)
 // if it's a new role, or the version is greater than the current version
 // for the role. Otherwise an error is returned.
 func (rdb RethinkDB) UpdateCurrent(gun data.GUN, update MetaUpdate) error {
-	now := time.Now()
-	checksum := sha256.Sum256(update.Data)
-	file := RDBTUFFile{
-		Timing: rethinkdb.Timing{
-			CreatedAt: now,
-			UpdatedAt: now,
-		},
-		GunRoleVersion: []interface{}{gun, update.Role, update.Version},
-		Gun:            gun.String(),
-		Role:           update.Role.String(),
-		Version:        update.Version,
-		SHA256:         hex.EncodeToString(checksum[:]),
-		Data:           update.Data,
+	// empty string is the zero value for tsChecksum in the RDBTUFFile struct.
+	// Therefore we can just call through to updateCurrentWithTSChecksum passing
+	// "" for the tsChecksum value.
+	if err := rdb.updateCurrentWithTSChecksum(gun.String(), "", update); err != nil {
```

Comment:
Why wouldn't we want to keep around the TS checksum for other lookup functionality?

Reply:
This just DRYs out code, I don't think it changed functionality. Did I miss something?

Reply:
ah, yes you're right that no functionality has changed.  `TSchecksum` was omitted from this write